### DISCUSSION TITLE
Hydrogen and jack2 on windows

### DISCUSSION THREAD

**Discussion by sirjohndfox** [hydrogen-music/hydrogen#1189] [Hydrogen on Windows]
I download the latest windows binary but seem only to be able to use port audio driver.  Possible to use jack on windows?  I want to try use hydrogen with protools by sending mmc and clock from protools to hydrogen

**Comment by theGreatWhiteShark** [comment]
Yes, version 1.0.1 on Windows was not built with Jack support (see https://github.com/hydrogen-music/hydrogen/issues/965)
